agent/exploits/cve_2014_7169.py

```diff
@@ -60,38 +70,5 @@ def check(self, target: definitions.Target) -> list[definitions.Vulnerability]:
             if elapsed < delay or elapsed - delay > MAX_DELAY_DIFFERENCE:
                 return []
 
-        vulnerability = self._generate_vulnerability_object(target.url)
+        vulnerability = self._create_vulnerability(target)
```

Comment:
This method can't be private. If we are using it this way, it should be public (and have unit tests).